client/components/notice/style.scss

```diff
@@ -137,15 +137,10 @@
 		font-weight: 600;
 
 		// Medium text underline hack
-		background-image: linear-gradient(to bottom, rgba(0,0,0,0) 75%, #fff 50%);
+		background-image: linear-gradient(to bottom, rgba(0,0,0,0) 50%, #fff 50%);
```

Comment:
This won't produce a single pixel line on hidpi screens. If we can't do that, we might as well go with `text-decoration: underline`